pom.xml

```diff
@@ -9,7 +9,7 @@
 
     <groupId>org.n52.sensorweb-server.sta</groupId>
     <artifactId>sta-parent</artifactId>
-    <version>3.0.3-SNAPSHOT</version>
+    <version>3.0.4-PR.1-SNAPSHOT</version>
```

Comment:
Next iteration is still 3.0.3-SNAPSHOT.

Reply:
Ja das maven:prepare wollte nicht so wie ich das will das hat den SNAPSHOT string einfach nicht mehr akzeptiert. Ändere ich noch :)